1.	Jiang S#, Qian Q#, Zhu T#, **Zong W**, Shang Y, Jin T, Zhang Y, Chen M, Wu Z, Chu Y, Zhang R, Luo S, Jing W, Zou D, Bao Y*, Xiao J*, Zhang Z*. Cell Taxonomy: a curated repository of cell types with multifaceted characterization. Nucleic Acids Research, 2023, 51: D853-D860.
2.	Kang H#, Zhu X, Cui Y, Xiong Z, **Zong W**, Bao Y*, Jia P*. A comprehensive benchmark of transcriptomic biomarkers for immune checkpoint blockades. Cancers, 2023, 15: 4094. doi: 10.3390/cancers15164094.
3.	Wang G#, Xiong Z#, Yang F, Zheng X, **Zong W**, Li R*, Bao Y*. Identification of COVID-19-associated DNA methylation variations by integrating methylation array and scRNA-Seq data at cell-type resolution. Genes, 2022, 13: 1109. doi: 10.3390/genes13071109.
4.	Wu S#, Huang Y#, Zhang M, Gong Z, Wang G, Zheng X, **Zong W**, Zhao W, Xing P, Li R*, Liu Z*, and Bao Y*. ASCancer Atlas: a comprehensive knowledgebase of alternative splicing in human cancers[J]. Nucleic Acids Research, 2023, 51: D1196-D1204.
5.	Xiong Z#, Yang F#, Li M#, Ma Y, Zhao W, Wang G, Li Z, Zheng X, Zou D, **Zong W**, Kang H, Jia Y, Li R*, Zhang Z*, Bao Y*. EWAS Open Platform: integrated data, knowledge and toolkit for epigenome-wide association study. Nucleic Acids Research, 2022, 50: D1004-D1009.
6.	Zhang Y#, Zou D#, Zhu T#, Teng X#, Chen M#, Niu G, **Zong W**, Pan R, Jing W, Sang J, Liu C, Zhao W*, Xiao J*, Bao Y*, Hao L*, Zhang Z*. Gene Expression Nebulas (GEN): a comprehensive data portal integrating transcriptomic profiles across multiple species at both bulk and single-cell levels. Nucleic Acids Research, 2022, 50: D1016-D1024.
7.	Wang H#, Zhang J#, Cheng Z, Si J, Wang T, Zhang X, Zhang H, **Zong W**, Liu L, Huang H, El-Kassaby A. *, Zhang D*, Xie J*. C-termini diversification of plant Growth-regulating factors evolved novel phenotypic effects on morphological traits. (Under review)
